lib/Runtime/Base/WindowsGlobalizationAdapter.cpp

```diff
@@ -385,7 +385,6 @@ namespace Js
 
         // Construct HSTRING of timeZoneId passed
         IfFailThrowHr(GetWindowsGlobalizationLibrary(scriptContext)->WindowsCreateStringReference(timeZoneId, static_cast<UINT32>(wcslen(timeZoneId)), &timeZoneHeader, &timeZone));
-        Assert(timeZone);
```

Comment:
Validity of timeZone value is decided by `windows.globalization.dll` and would like to stick to that. If this timeZone was invalid because of empty string or invalid value, `hr = timeZoneCalendar->ChangeTimeZone(timeZone);`  would return error hr in which case we will return.
However, I can go back and check other places where we might remove similar assert.